.github/workflows/main.yml

```diff
@@ -148,6 +149,12 @@ jobs:
                 git fetch origin $DOCS_BRANCH
                 git checkout -b $DOCS_BRANCH
                 git status
+                git clone --filter=blob:none --no-checkout https://github.com/tulip-control/tulip-control.git temporary
```

Comment:
As far as I know, this is the fastest and smallest way to get the repository history such that we can compare commit hashes. The `filter` switch here causes a [partial clone](https://git-scm.com/docs/partial-clone) that excludes file contents.